src/Handler/Exception/ExceptionResolver.php

```diff
@@ -25,6 +25,12 @@ public static function resolveException(\Throwable $e): \Throwable
                 if (strpos($e->getMessage(), 'No more room in database') !== false) {
                     return NoSpaceException::createForFullDB($e);
                 }
+                if (strpos($e->getMessage(), "available data bytes in the table's perm row has exceeded")) {
```

Comment:
kdyz budeme predpokladat, ze driver si ma take ocheckovat pocet sloupcu, tak za me to nema delat retrospektivne, ale checkem pred pustenim query. a odpoved by imho mela byt odpovidajici s `ColumnCollection`